api/src/changeRequests/processChangeRequest.ts

```diff
@@ -80,37 +80,38 @@ export async function processChangeRequest(
     // Insert / update the document in the database
     const upsertResult = await db.upsertDoc(doc);
 
+    // ******************* tmp disable changeDoc, until it will be implimented on the CMS (ATM the changeDoc's only slow down the api) ********************
     // Generate change document and store in database
-    if (upsertResult.changes) {
-        const changeDoc = new ChangeDto();
-        changeDoc._id = "change:" + randomUUID();
-        changeDoc.type = DocType.Change;
-        changeDoc.docId = doc._id;
-        changeDoc.docType = doc.type;
-        changeDoc.updatedTimeUtc = upsertResult.updatedTimeUtc;
-        changeDoc.changes = upsertResult.changes;
-        changeDoc.changedByUser = userId;
-
-        // Apply the group membership or ACL of the included document to the change document
-        if (upsertResult.changes.memberOf) {
-            changeDoc.memberOf = upsertResult.changes.memberOf;
-        }
-        if (upsertResult.changes.acl) {
-            changeDoc.acl = upsertResult.changes.acl;
-        }
-
-        // Apply the parentId of the included document to the change document
-        if (upsertResult.changes.parentId) {
-            changeDoc.parentId = upsertResult.changes.parentId;
-        }
-
-        // Apply the language of the included document to the change document
-        if (upsertResult.changes.language) {
-            changeDoc.language = upsertResult.changes.language;
-        }
-
-        return db.insertDoc(changeDoc);
-    }
+    // if (upsertResult.changes) {
```

Comment:
Related to the previous comment, how sure are we that this will be changed soon? If not, I would suggest to remove this code rather than comment it out, so it doesn't keep laying around if we never get to actually solving this problem. We can always easily find the code back in Git/Github, especially if you create an issue for solving this that then can link to this PR.